### ISSUE TITLE
Please make hashtags normal links

### ISSUE THREAD

**Issue by ungoldman** [bluesky-social/social-app#3052]
**Is your feature request related to a problem? Please describe.**

When I right click on a hashtag link on the web version of bluesky, I can't copy the link or open the link in a new tab, as it is not a link. This makes me sad.

**Describe the solution you'd like**

When I right click on a hashtag link on the web version of bluesky, I can copy the link or open the link in a new tab, as it is a link. This makes me happy.

**Describe alternatives you've considered**

The `TagMenu` context menu behavior is an interesting addition that affords the user lot of choices right away. However I think not allowing the hashtag element to be rendered as a normal link (`<a>` tag) in the web interface is a miss. I'd like to humbly request that the hashtag be rendered as a link to allow easily opening the link in a new window, or copying it for pasting elsewhere (generally following the time-honored maxim of "don't break the web").

There's an argument to be made that hashtags should _only_ be links, and instead of a context menu, the search page would detect that it's rendering hashtag results and show relevant contextual actions like filtering by user or muting the hashtag. This would mean less clicks for a user to interact with a hashtag in the default way. However I'm sure this went through some design and UX iterations and I don't want to yuck anyone's yum if the hashtag context menu is a prized possession.

**Additional context**

![Screenshot 2024-02-29 at 10 40 05](https://github.com/bluesky-social/social-app/assets/427322/7cae7ad5-6bcf-4c19-a683-4ab20fac4753)

**Addendum**

Congrats on the newly shipped hashtag functionality!

Good luck and happy coding :v:

**Comment by HeyItsLollie** [comment]
As a side-note, it seems context menu options are also not links, and also cannot be opened into a new tab. This also makes me sad!

**Comment by mschwendener** [comment]
> When I right click on a hashtag link on the web version of bluesky, I can't [...] open the link in a new tab

I'm missing this, too. 

---

I can send somebody a link to a certain hashtag, for example https://bsky.app/search?q=igelu2024 , but the sort is not part of the URL. What I would like to do is link directly to "Latest". 

Thank you :-)

**Comment by xPaw** [comment]
I agree, the hashtags should be a normal <a> tag so that right click and middle click work on them. You can still keep the dropdown menu when left clicking them.

**Comment by mozzius** [comment]
Works now I think
